src/reuse/helper/errorHandler.ts

```diff
@@ -6,34 +6,45 @@ export interface IErrorHandler {
 }
 
 export class CustomError extends Error {
-  constructor(message: string, name: string, stack?: string) {
+  constructor(message: string, displayStack: boolean) {
     super(message);
+    this.name = this.constructor.name;
     this.message = message;
-    this.name = name;
-    this.stack = stack;
+    this.stack = displayStack ? this._getFormattedStackTrace(this.stack) : "";
+  }
+
+  // =================================== HELPER ===================================
+  private _getFormattedStackTrace(stack?: string): string {
+    if (stack) {
+      var stackTrace = stack
+        .split("\n")
+        .map((line: string) => line.replace(/\s+at\s+/, ""))
+        .join("\n");
+      return stackTrace;
+    } else {
+      return "";
+    }
   }
```

Comment:
Why can't we keep the stack as is, instead of removing the "at"s?

Reply:
You can probably remove the first line from the stack - ErrorHandler.logException

Reply:
I agree with Herald

Reply:
@heraldantony  now keeping "at" in each line its not removing it

Reply:
@heraldantony first line also removed